.github/workflows/main.yml

```diff
@@ -0,0 +1,35 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  main:
+    runs-on: ubuntu-18.04
+
+    steps:
+    - uses: actions/checkout@v2
+      # Create a master branch for commitlint
+      # https://github.com/conventional-changelog/commitlint/issues/6
+    - run: git remote set-branches origin master && git fetch
+    - uses: actions/setup-node@v1
+      with:
+        node-version: '12.x'
+    - name: Install required packages
+      # add required dependencies for webkit, chromium and firefox (playwright)
+      run: |
+        sudo apt-get update
+        sudo apt-get install libwoff1 libopus0 libwebp6 libwebpdemux2 libenchant1c2a libgudev-1.0-0 libsecret-1-0 libhyphen0 libgdk-pixbuf2.0-0 libegl1 libnotify4 libxslt1.1 libevent-2.1-6 libgles2 libgl1 libvpx5
+        sudo apt-get install libnss3 libxss1 libasound2
+        sudo apt-get install libdbus-glib-1-2 libxt6
+    - name: Setup
+      env:
+        NODE_ENV: development
+      run: yarn
+      # TODO: cache node_modules and re-use between workflows
```

Comment:
You can use https://github.com/actions/cache for this

Reply:
Good call !!! and find, exactly what I was looking for. Thank you for suggesting

Reply:
Added 927744a